suggestions.md

```diff
@@ -0,0 +1,228 @@
+        "score": "9",
+        "notes": "some notes, eventually empty or null"
+      },
+      "educationYearEvidence": {
+        "score": "3.6",
+        "notes": "some notes, eventually empty or null"
+      },
+      "personTypeEvidence": {
+        "score": "4",
+        "notes": "some notes, eventually empty or null"
+      },
+      "articleCountEvidence": {
+        "score": "6.7",
+        "notes": "some notes, eventually empty or null"
+      },
+      "averageClusteringEvidence": {
+        "score": "7",
+        "notes": "some notes, eventually empty or null"
+      }
+    },
+    "metadata": {
+        "dc.identifier.uri": [
+          {
+            "value": "https://publication/0000-0003-3681-2038",
+            "language": null,
+            "authority": null,
+            "confidence": -1,
+            "place": -1
+          }
+        ],
+        "dc.title" : [
+          {
+            "value" : "publication one",
+            "language" : null,
+            "authority" : null,
+            "confidence" : -1
+          }
+        ],
+        "dc.date.issued" : [
+          {
+            "value" : "2010-11-03",
+            "language" : null,
+            "authority" : null,
+            "confidence" : -1
+          }
+        ]
+    },
+    "type": "suggestion",
+    "_links": {
+      "target": {
+        "href": "https://dspace7.4science.cloud/server/api/core/items/gf3d657-9d6d-4a87-b905-fef0f8cae26"
+      },
+      "self": {
+        "href": "https://dspace7.4science.cloud/server/api/integration/suggestions/reciter:gf3d657-9d6d-4a87-b905-fef0f8cae26:24694772"
+      }
+    }
+}
+```
+
+## Search methods
+### Get suggestions by a given target
+**/api/integration/suggestions/search/findByTargetAndSource?target=:target-uuid&source=:source-key[&size=10&page=0]**
+
+It returns the list of suggestions and corresponding evidence from a specific source for the requested individual/target
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination). Data can be sorted by trust desc (default) or asc. Please note that the sort parameter must use the word trust instead than score for the sorting as score have a reserved meaning 
+* target: mandatory, the uuid associated with your target profile
+* source: mandatory, the key of the source that you want to query for suggestion
+
+Return codes:
+* 200 OK - if the operation succeed
+* 400 Bad Request - if the uuid parameter is missing or invalid
+
+sample for a search /server/api/integration/suggestions/search/findByTargetAndSource?target=gf3d657-9d6d-4a87-b905-fef0f8cae26c&source=reciter
+```json
+{
+  "_embedded": {
+    "reciterSourceEntries": [
+{
+        "id": "24694772",
+        "display": "publication one",
+        "source": "reciter",
+        "external-source-uri": "https://dspace7.4science.cloud/server/api/integration/reciterSourcesEntry/pubmed/entryValues/24694772",
+        "evidences": {
+          "acceptedRejectedEvidence": {
```

Comment:
@frabacche : I understand they are configurable.  However, the REST Contract is supposed to provide examples that are realistic based on DSpace's current features.  I feel listing a large number of evidences that _don't exist in DSpace_ is misleading.  This list should at least be updated to show the few that _currently exist_ and then maybe have **one example** of how it might look if extended.